x-pack/plugins/security_solution/public/timelines/components/timeline/body/translations.ts

```diff
@@ -16,7 +16,7 @@ export const NOTES_TOOLTIP = i18n.translate(
 export const NOTES_DISABLE_TOOLTIP = i18n.translate(
   'xpack.securitySolution.timeline.body.notes.disableEventTooltip',
   {
-    defaultMessage: 'Add notes for event filtered by a timeline template is not allowed',
+    defaultMessage: 'Notes may not be added here while editing a template timeline',
```

Comment:
While testing locally in the regular (non-template) mode, I noticed, per the screenshot below, that the tooltip currently reads `Add or view notes for this event`:

<img width="383" alt="add-or-view-notes" src="https://user-images.githubusercontent.com/4459398/87101257-da3bc380-c20b-11ea-8fda-f8776fbf6cfe.png">

I don't think the `or view` part of the tooltip is valid, because clicking the button doesn't view notes. Notes are always displayed inline, per the screenshot below:

<img width="463" alt="inline-note" src="https://user-images.githubusercontent.com/4459398/87101970-b11c3280-c20d-11ea-9f26-1f4302c400c8.png">

Would you be willing to change `NOTES_TOOLTIP` in `translations.ts` from `Add or view notes for this event` to `Add notes for this event`?